template/main.js

```diff
@@ -362,6 +367,14 @@ function init($, _, locale, Handlebars, apiProject, apiData, Prism, sampleReques
                     };
                 }
 
+                if (apiProject.sampleUrl == false) {
```

Comment:
this should probably check if the value is undefined, not falsy? it breaks generation in cases where sampleUrl is explicitly set to false now

Reply:
my bad, didn't see #929 and #930, but `useHostUrlAsSampleUrl: true` doesn't seem to be doing anything for me

Reply:
I don't think the parameter is yet available in the release.

Reply:
@matmen use the `dev` branch!